### PR TITLE
GH2917 - Add note on accepted PRs in code and metadata for template

### PR DESCRIPTION
**Pull request by lorikrell**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [X] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will...

Add a note to the code contributing guide regarding accepted pull requests for 2.2/2.3.
Add a metadata section to the basic template, including the new group for toc.

## Additional information

Resolves https://github.com/magento/devdocs/issues/2917

whatsnew
Added announcements and supported pull requests for [v2.1](https://devdocs.magento.com/guides/v2.1/contributor-guide/contributing.html), [v2.2](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html), and [v2.3](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html) Contributing guide. Updated the [basic template](https://devdocs.magento.com/guides/v2.2/contributor-guide/templates/basic_template.html) for DevDocs contributions.

### CODE REVIEW

**Comment by lorikrell** [comment]
@dshevtsov  If you want to verify the metadata for the basic template. 
Community is reviewing the other note in contributing.md.

File: guides/v2.1/contributor-guide/templates/basic_template.md [hunk 1]
```diff
@@ -12,6 +12,23 @@ The purpose of this page is to provide you with a pre-formatted template and use
 
 You can start off by editing the local version of this file using markdown language (and {% glossarytooltip a2aff425-07dd-4bd6-9671-29b7edefa871 %}HTML{% endglossarytooltip %} where needed). Then, create a Pull Request to have your contribution reviewed by the DevDocs team.
 
+## Metadata parameters
+
+Add the following {% glossarytooltip 3f0f2ef1-ad38-41c6-bd1e-390daaa71d76 %}metadata{% endglossarytooltip %} parameters at the top of your page. We use this data when generating the DevDocs for the following:
+
+Parameter | Description
+--- | ---
+`group:` | The table of contents this file belongs to. Only add the name of the file without the extension. For example, the group of this file is `group: contributor-guide`.
```
Comment (dshevtsov) [inline guides/v2.1/contributor-guide/templates/basic_template.md]: Add path to the directory with toc files: `_data/toc`
And extend the example:
_the group of this file is `group: contributor-guide`, it points to the table of contents defined in `_data/toc/contributor-guide.yml`_

File: guides/v2.1/contributor-guide/templates/basic_template.md [hunk 2]
```diff
@@ -12,6 +12,23 @@ The purpose of this page is to provide you with a pre-formatted template and use
...
+`title:` | The title of the page.
+`ee_only:` | Optional. If set to `true`, graphics/cues display on the page indicating it applies to {{site.data.var.ee}}.
```
Comment (dshevtsov) [inline guides/v2.1/contributor-guide/templates/basic_template.md]: add descriptions of `functional_areas` and `redirect_from` to the table

File: guides/v2.1/contributor-guide/templates/basic_template.md [hunk 3]
```diff
@@ -12,6 +12,23 @@ The purpose of this page is to provide you with a pre-formatted template and use
 
 You can start off by editing the local version of this file using markdown language (and {% glossarytooltip a2aff425-07dd-4bd6-9671-29b7edefa871 %}HTML{% endglossarytooltip %} where needed). Then, create a Pull Request to have your contribution reviewed by the DevDocs team.
 
+## Metadata parameters
+
+Add the following {% glossarytooltip 3f0f2ef1-ad38-41c6-bd1e-390daaa71d76 %}metadata{% endglossarytooltip %} parameters at the top of your page. We use this data when generating the DevDocs for the following:
```
Comment (dshevtsov) [inline guides/v2.1/contributor-guide/templates/basic_template.md]: I don't see _metadata_ in the [Glossary](https://docs.magento.com/m2/ce/user_guide/backmatter/glossary.html)
Remove `glossarytooltip`

Comment (lorikrell) [reply guides/v2.1/contributor-guide/templates/basic_template.md]: Removed. Also removed it from the tutorial first page file (new in changed files for this PR).

File: guides/v2.1/contributor-guide/templates/basic_template.md (hunk 2 above)
Comment (lorikrell) [reply guides/v2.1/contributor-guide/templates/basic_template.md]: Added to basic template and tutorial first page file (new in changed files for this PR).

File: guides/v2.1/contributor-guide/templates/basic_template.md (hunk 1 above)
Comment (lorikrell) [reply guides/v2.1/contributor-guide/templates/basic_template.md]: Updated with the additions. I didn't add this to the tutorial template as it uses different group functions for that toc.

File: guides/v2.1/contributor-guide/templates/basic_template.md [hunk 4]
```diff
@@ -14,12 +14,14 @@ You can start off by editing the local version of this file using markdown langu
...
+Add the following metadata parameters at the top of your page. We use this data when generating the DevDocs for the following:
 
 Parameter | Description
 --- | ---
-`group:` | The table of contents this file belongs to. Only add the name of the file without the extension. For example, the group of this file is `group: contributor-guide`.
+`group:` | The table of contents this file belongs to, which points to files located in `_data/toc/`. Only add the name of the file without the extension. For example, the group of this file is `group: contributor-guide`, which points to the table of contents defined in `_data/toc/contributor-guide.yml`.
 `title:` | The title of the page.
+`functional_areas:`  |  Optional. Adds facets for search results. Available facets include: sales, products, carts, customers, marketing, account, content, reports, stores, system, catalog, orders, frontend, theme, staging, search, configurations, integration, services, tools, setup, testing or test, standards, install, upgrade, B2B, cloud, and bundled extensions.
```
Comment (dshevtsov) [inline guides/v2.1/contributor-guide/templates/basic_template.md]: The facets are case sensitive.
Update the list with the exact capitalization as defined in our internal wiki list.

File: guides/v2.1/contributor-guide/templates/basic_template.md [hunk 5]
```diff
@@ -14,12 +14,14 @@ You can start off by editing the local version of this file using markdown langu
 
 ## Metadata parameters
 
-Add the following {% glossarytooltip 3f0f2ef1-ad38-41c6-bd1e-390daaa71d76 %}metadata{% endglossarytooltip %} parameters at the top of your page. We use this data when generating the DevDocs for the following:
+Add the following metadata parameters at the top of your page. We use this data when generating the DevDocs for the following:
```
Comment (dshevtsov) [inline guides/v2.1/contributor-guide/templates/basic_template.md]: Add that we use YAML for the metadata in front matter.

Comment (lorikrell) [reply guides/v2.1/contributor-guide/templates/basic_template.md]: Added note to the opening text.

File: guides/v2.1/contributor-guide/templates/basic_template.md (hunk 4 above)
Comment (lorikrell) [reply guides/v2.1/contributor-guide/templates/basic_template.md]: Added that info, and verified all entries match the list.

**Comment by lorikrell** [comment]
Adding new notes for engcom. Targeting merge for Monday Sept 17th.

File: _includes/contributor/2-1-end.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+{:.bs-callout .bs-callout-warning}
+**v2.1 End of Contribution:**
+We are no longer accepting contributions to the v2.1 release line. Please submit contributions for the [v2.2](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html) and [v2.3](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html) release lines. We made this change to increase focus, issue investigation, and pull request approvals for v2.2 and v2.3 release lines.
```
Comment (hguthrie) [inline _includes/contributor/2-1-end.md]: Maybe change tense to "We no longer accept..."

Comment (lorikrell) [reply _includes/contributor/2-1-end.md]: Thanks updated, made similar change to 2.2 notice.

**Comment by lorikrell** [comment]
Gathering feedback from EngCom for any further changes from their side. Target for publication Sept 17th.

File: _includes/contributor/2-1-end.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+{:.bs-callout .bs-callout-warning}
+**v2.1 End of Contribution:**
+As of September 15, 2018, we no longer accept contributions to the v2.1 release line. Please submit contributions for the [v2.2](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html) and [v2.3](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html) release lines. We made this change to increase focus on issue investigations and pull request approvals for v2.2 and v2.3 release lines.
```
Comment (shrielenee) [inline _includes/contributor/2-1-end.md]: I would take "Please" out. Something like "Continue submitting contributions for the ..."

"We made this change to increase our focus.."

File: _includes/contributor/2-2-submission.md [hunk 8]
```diff
@@ -0,0 +1,3 @@
+{:.bs-callout .bs-callout-info}
+**v2.2 Contribution Notice:**
+As of September 15, 2018, to keep 2.2 and 2.3 releases aligned, we only accept pull requests to 2.2-develop that contain changes already existing in 2.3-develop. We recommend submitting pull requests to 2.3-develop then [back porting](#porting) the changes to 2.2-develop.
```
Comment (shrielenee) [inline _includes/contributor/2-2-submission.md]: Maybe "As of September 15, 2018, in an effort to keep v2.2 and v2.3 releases aligned, we will only accept pull requests..."

File: _includes/contributor/2-3-submission.md [hunk 9]
```diff
@@ -0,0 +1,3 @@
+{:.bs-callout .bs-callout-info}
+**v2.3 Contribution Notice:**
+As of September 15, 2018, to keep 2.2 and 2.3 releases aligned, we accept pull requests to 2.3-develop and encourage [back ports](#porting) to 2.2-develop. We no longer accept pull requests to the v2.1 or v2.0 release lines to focus all development efforts on v2.3 and v2.2.
```
Comment (shrielenee) [inline _includes/contributor/2-3-submission.md]: "in an effort to keep v2.2 and v2.3 releases aligned,"

File: _includes/contributor/2-1-end.md (hunk 7 above)
Comment (lorikrell) [reply _includes/contributor/2-1-end.md]: Updated!

File: _includes/contributor/2-2-submission.md (hunk 8 above)
Comment (lorikrell) [reply _includes/contributor/2-2-submission.md]: Updated, thanks!

File: _includes/contributor/2-3-submission.md (hunk 9 above)
Comment (lorikrell) [reply _includes/contributor/2-3-submission.md]: Changed, thanks!

**Pull request merged**